### PR TITLE
Support V3Det: Vast Vocabulary Visual Detection Dataset (ICCV 2023 Oral)

### PR DESCRIPTION
**Pull request by V3Det**
## Motivation
Support V3Det: Vast Vocabulary Visual Detection Dataset (ICCV 2023 Oral).

## Modification
1. Support V3Det dataset
2. Support multi-processing evaluation (It takes hours to evaluate 13204 categories of V3Det with original single-processing evaluation.)
3. CrossEntropyCustomLoss and FocalCustomLoss with custom_activation and custom_accuracy to better monitor the training process of V3Det
4. An option to permute repeated GT bboxes in the max_iou_assigner. Defaults to None.
5. An option of FCOS with cls_predictor_cfg. Defaults to None.
6. An option to force each level of FPN to have gradients to avoid potential deadlock. Defaults to False.
7. ReadMe and Configs.

### CODE REVIEW

File: configs/v3det/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<p>
+<div align="center">
+
+# <img src="v3det_icon.jpg" height="25"> V3Det: Vast Vocabulary Visual Detection Dataset
```
Comment (RangeKing) [inline configs/v3det/README.md:4]: the same modification as "introduction.jpg"

Comment (hhaAndroid) [reply configs/v3det/README.md:4]: 这个图标很小，算了，不动了

**Review by hhaAndroid — APPROVED** [review]
